src/main/java/com/dev/cinema/config/AppConfig.java

```diff
@@ -14,9 +14,7 @@
 @Configuration
 @PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
-        "com.dev.cinema.dao",
-        "com.dev.cinema.security",
-        "com.dev.cinema.service"
+        "com.dev.cinema",
```

Comment:
let's add only those packages which we need to Scan.

Reply:
done